src/server/api/routes/events.router.js

```diff
@@ -137,4 +137,56 @@ router.delete('/:userId/:eventId', (req, res) => {
     .catch((error) => console.log(error));
 });
 
+// edit an event
+
+/**
+ * @swagger
+ * /events/{EID}/{UID}:
+ *  patch:
+ *    summary: Edit an event
+ *    description:
+ *      Will edit an event.
+ *    produces: application/json
+ *    parameters:
+ *      - in: path
+ *        name: EID, UID
+ *        description: EID is the event ID to patch, and UID is the User ID to edit
+ *      - in: body
+ *        name: event
+ *        description: The event to edit.
+ *        schema:
+ *          type: object
+ *          properties:
+ *            title:
+ *              type: string
+ *            startDate:
+ *              type: string
+ *              format: date-time
+ *            endDate:
+ *              type: string
+ *              format: date-time
+ *            classId:
+ *              type: string
```

Comment:
Are you allowing editing of all these properties? They should match the last part of the functionality in the other file.

Reply:
only venue editable, removing all unnecessary